### ISSUE TITLE
Learning rate decay settings for CUB

### ISSUE THREAD

**Issue by Pangoraw** [MinaGhadimiAtigh/Hyperbolic-Busemann-Learning#2]
Hi!

I was wondering what is the LR decay setting used for the CUB training runs in the paper?
LR decay was used in CIFAR10 and CIFAR100 for epochs 1000 and 1100; but there are no informations about the decay setting for CUB.

I tried using a decay at epochs 2000 and 2100 and the accuracy seemed better than what was reported in the paper (Test accuracy at 53.54% for dim=50 and 41.8% at dim=3).

Thank you for the great paper!
